homeassistant/components/nyt_games/strings.json

```diff
@@ -0,0 +1,29 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "token": "Token"
+        },
+        "data_description": {
+          "token": "The NYT Games NYT-S cookie value."
```

Comment:
If I remember well descriptions only works for selectors but you use `str` in the config flow schema

Reply:
![image](https://github.com/user-attachments/assets/d77904b9-986d-4e3d-b776-b1041f4d45a3)